src/vehicle_use.cpp

```diff
@@ -1065,7 +1066,7 @@ bool vehicle::start_engine( const int e )
     if( sfx::has_variant_sound( "engine_start", eng.info().get_id().str() ) ) {
         sfx::play_variant_sound( "engine_start", eng.info().get_id().str(),
                                  eng.info().engine_noise_factor() );
-    } else if( is_engine_type( e, fuel_type_muscle ) ) {
+    } else if( einfo.fuel_type == fuel_type_muscle ) {
```

Comment:
Using "einfo.fuel_type == fuel_type_muscle" was better for testing because "is_engine_type" function does not always work.